primer/brand
# Fix: SubdomainNavBar and Radio component bugs in safari (iOS)

## Summary

Fixed mobile menu bug in safari iOS
Fixed radio outline bug in safari iOS

## List of notable changes:



- **updated** the animation-specific classes to not apply on mobile, where visibility/invisibility through interesection observer is not required.
- **added** a new hook to detect window size on the client
- **added** a new story for custom backgrounds on Radio components
- **added** a new pseudo class implementation for the outline, which doesn't work on Safari

## Steps to test:



1. Try this URL on mobile on an iphone with Safari 
1. Open the menu
1. Check menu items load correctly
1. Open this URL in your browser in Safari
1. Verify there's no outline on the radio anymore

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos


https://user-images.githubusercontent.com/13340707/200811895-9cd40a97-195e-4cae-98c8-174546842507.mov



<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">


![radio with a square outline where border radius did not correctly apply](https://user-images.githubusercontent.com/13340707/200811719-88462d0d-4f60-4fda-94ea-b0858b7bb179.png)

 </td>
<td valign="top">


![radio with a rounded border to replace the outline](https://user-images.githubusercontent.com/13340707/200811705-68f9cd88-9d6e-4704-a83c-4d5af31c89f7.png)

</td>
</tr>
</table>
